### ISSUE TITLE
Longer training and stronger augmentation do not work for Cifar 100

### ISSUE THREAD

**Issue by Jun-CEN** [sgvaze/osr_closed_set_all_you_need#22]
Hi,

I am trying to use the RandAug instead of RandCrop and cosine learning rate schedule to train the whole Cifar100 dataset. But they do not work. The baseline is RandCrop with step learning rate schedule (initial learning rate is 0.1 and divided by 2 at [60, 120, 160] epochs with 300 epochs in total). The closed-set accuracy of baseline is 0.75, and RandAug and cosine learning rate schedule (restart 0 or 2 times for 600 epochs) does not work, can you give me some advise?

Best regards.

**Comment by sgvaze** [comment]
Hello, apologies for the (very) late response!

We performed all ablations on TinyImageNet, taking the best setting for this and transferring it to the other tasks. If I recall, we only tuned the RandAugment hyper-parameters for each of the other datasets. 

As a rule of thumb, best OSR performance will be given by optimising hyper-parameters for the test (or val) closed-set accuracy of the dataset you are working with.
